src/test/js/multibranch/multibranchOpening.js

```diff
@@ -0,0 +1,72 @@
+/** @module multibranchOpening
+ * @memberof multibranch
+ * @description TEST: basic tests around multiBranch and encoding of branches
+ */
+const async = require('async');
+const path = require('path');
+const pathToRepo = path.resolve('./target/test2-project-folder');
+const sourceRep = './src/test/resources/multibranch_2';
+const git = require('../../../main/js/api/git');
+const pageHelper = require('../../../main/js/util/pageHelper');
+const createCallbackWrapper = pageHelper.createCallbackWrapper;
+const sanityCheck = pageHelper.sanityCheck;
+
+module.exports = {
+  // ** creating a git repo */
+  before: function (browser, done) {
+    // we creating a git repo in target based on the src repo (see above)
+    git.createRepo(sourceRep, pathToRepo)
+      .then(function () {
+        git.createBranch('feature/1', pathToRepo)
+          .then(done);
+      });
+  },
+  /**
+   * Make sure we can open the master and the feature/1 branch results screen from activity and branch tab
+   * Regression: https://issues.jenkins-ci.org/browse/JENKINS-40027
+   */
+  'open master and feature/1 branch from activity and from branch tab': function (browser) {
+    const cases = [{
+        regexp: 'tr[id^="master"]',
+        name: 'masterActivityMB',
+      }, {
+        regexp: 'tr[id^="feature"]',
+        name: 'featureActivityMB',
+      },
+      {
+        regexp: 'tr[id^="master"]',
+        name: 'masterBranchesMB',
+        branch: true,
+      }, {
+        regexp: 'tr[id^="feature"]',
+        name: 'featureBranchesMB',
+        branch: true,
+      },
+    ];
+    async.mapSeries(cases, function (usecase, callback) {
```

Comment:
this is where most of the test happens, but now is an obscure named map. This is not friendly for others trying to follow the pattern. Its clever, and great production code, but it isn't an acceptance test